core/tests/protractor/editorFeatures.js

```diff
@@ -560,6 +560,7 @@ describe('Suggestions on Explorations', function() {
     users.login('user1@ExplorationSuggestions.com');
     creatorDashboardPage.get();
     creatorDashboardPage.navigateToExplorationEditor();
+    general.waitForSystem();
```

Comment:
Hi @bansalnitish! This passes all tests and looks good but I am not very sure if this is the best way to resolve the flakiness here. 
(Sorry for such a late review!)

Reply:
Let's wait to hear from other people of the community. Thanks!

Reply:
Hi @apb7 Sometimes the flakiness is fixed due to a wait statement. If you see the test running in selenium (locally), sometimes the test fails if you try to click a button or access an element before the page has loaded completely. So I think this might be a reasonable fix for the flake.

Reply:
This seems like a fine fix, though is there a way to ensure that the system doesn't wait forever?

Reply:
@kevinlee12 If you check the function, general.waitForSystem() introduces a delay of four seconds for allowing whatever computation is happening to catch up.

Reply:
sgtm, though it should be documented that this will introduce a wait for up to 4 seconds.

Reply:
@bansalnitish Do you think you could make the fix? Thanks!

Reply:
I think it is fine since we use the same construct at many other places.

Reply:
In that case, no need to insert the comment.